### ISSUE TITLE
The generated motions are static?

### ISSUE THREAD

**Issue by LinghaoChan** [GuyTevet/MotionCLIP#5]
I train the model with the provided code. The generated results are almost static. What happened?

**Comment by GuyTevet** [comment]
Hi Ling-Hao,
Did you train with our configuration (in readme), or use your own settings?

**Comment by LinghaoChan** [comment]
> Hi Ling-Hao, Did you train with our configuration (in readme), or use your own settings?

I check the training and inference settings just now. The static results are inferenced by 500-epoch pretrained model. Motions are active when I switch to the pretrained model with 100-epoch training.

Are your results the same?

**Comment by GuyTevet** [comment]
Yup. We present results on epoch 100. Anyway, please note that the training loop currently has a reproduction issue (detailed in readme). Please reopen if you have any other questions regarding this.

**Comment by srph25** [comment]
Hi,
After retraining paper-model with the README configuration, I'm still getting constant movements.
After 100 epochs:
![clip_text_paper_texts_fig_100](https://user-images.githubusercontent.com/9195029/193869896-0c846815-2872-4079-8e51-06bf61f551c8.gif)
20 epochs works slightly better:
![clip_text_paper_texts_fig_020](https://user-images.githubusercontent.com/9195029/193871053-758d7618-e3ab-4231-a35e-1215cbac9ccd.gif)
Whereas loading the pretrained weights gives the expected results:
![original](https://user-images.githubusercontent.com/9195029/193870144-3ebad22f-46b3-48e5-ac52-fae926bfd220.gif)
Any ideas what I did wrong? Thank you!

**Comment by srph25** [comment]
Solved my previous post:
Turned out conda was loading my site-packages installed pytorch 1.10.2 instead of the repository suggested 1.8.1. Downgraded by uninstalling the more recent version.

Now I get much better loss values and closer results after 100 epochs:
![clip_text_paper_texts_fig_100](https://user-images.githubusercontent.com/9195029/193952735-b98b9485-fd37-4a92-b9af-6dbfa5623d1c.gif)

**Comment by shunlinlu** [comment]
Hi, I followed the script and code with torch 1.8.1 + cuda 11.1, but still got static motions on my own training on Amass. Official checkpoints work well. Any other clues here? @GuyTevet 
Is it possible to share your env info with me? Just wanna align the environment installation. @srph25 Thanks a lot.

**Comment by LinghaoChan** [comment]
According to similar problems met by @shunlinlu and @srph25 . I hope to reopen the issue to fix the bug~
